programs/util.h

```diff
@@ -118,9 +124,16 @@ U64 UTIL_getFileSize(const char* infilename);
 U64 UTIL_getTotalFileSize(const char* const * fileNamesTable, unsigned nbFiles);
 int UTIL_getFileStat(const char* infilename, stat_t* statbuf);
 int UTIL_setFileStat(const char* filename, stat_t* statbuf);
+int UTIL_getDirectoryStat(const char* infilename, stat_t* statbuf);
 int UTIL_chmod(char const* filename, mode_t permissions);   /*< like chmod, but avoid changing permission of /dev/null */
 int UTIL_compareStr(const void *p1, const void *p2);
 const char* UTIL_getFileExtension(const char* infilename);
+int   UTIL_isFileNameValidForMirroredOutput(const char *filename);
+void  UTIL_convertPathnameToDirName(char *pathname);
```

Comment:
Even though that's implied by the function signature,
please document that this function is mutating its argument.

Reply:
done.